selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/workspaces/ProjectStateAfterRenameWorkspaceTest.java

```diff
@@ -57,6 +60,11 @@ public void setUp() throws Exception {
     ide.open(workspace);
   }
 
+  @AfterClass
+  public void tearDown() throws Exception {
+    testWorkspaceServiceClient.delete(WORKSPACE_NEW_NAME, workspace.getOwner().getName());
```

Comment:
how many workspaces are used in this test? What are names?

Reply:
2 workspaces: default one which is created automatically, and second one which is created during execution of the test.